### DISCUSSION TITLE
Rule Chaining examples please

### DISCUSSION THREAD

**Discussion by DevRalphM** [microsoft/RulesEngine#202] [Q&A]
Thank you for this excellent Rules Engine library!!

I'm using this library in a business critical application for my employer. I've developed some business rules for the application 
 and written some until tests to prove everything works as required.

For a number of situations I need to use Rule Chaining, but there are no examples to learn from and rule chaining is not documented.

Could you release some of your unit tests you use to test the rule chaining functionality? I'm assuming that no documentation has been created, but not released.
Any help you could give or examples would be very helpful.

Many Thanks

**Comment by abbasc52** [comment]
@DevRalphM  basic example of Rule chaining is available now - https://microsoft.github.io/RulesEngine/#evaluaterule
